javascript/javascript1/week1/.history/ez-namey_20220526130828.js

```diff
@@ -0,0 +1,2 @@
+let firstWords = ["Easy", "Nice", "Amazing", "Cool", "Fantastic", "Innovative", "Revolutionary", "Futuristic", "AI-driven", "Successfull"]
```

Comment:
The whole `.history` folder should be added to a `gitignore` file or removed, because of it, it looks like you changed 56 files instead of 4. 
<img width="201" alt="Screenshot 2022-05-27 at 19 39 12" src="https://user-images.githubusercontent.com/10253713/170762809-4a30a80a-52b2-45d0-929f-f30feec4dffc.png">

Reply:
How to do it? I have added .history/* in gitignore file. It is because probably I have used  git add .

Reply:
You should delete the folder locally and push changes to github.